### PR TITLE
Fix case of component names in Modelica.Mechanics.MultiBody.Examples.Loops

### PR DESCRIPTION
**Pull request by beutlich**
Fix case of component names as reported by #2458.

### CODE REVIEW

File: Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo [hunk 1]
```diff
@@ -13,20 +13,20 @@ package Utilities "Utility models for Examples.Loops"
       "Offset of crank pin from center axis";
     parameter SI.Length crankPinLength=0.1
       "Offset of crank pin from center axis";
-    parameter SI.Angle cylinderInclination=0 "Inclination of cylinder";
+    parameter SI.Angle cylinderInclinationAngle=0 "Inclination of cylinder";
```
Comment (tobolar) [inline Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo]: Why this change?

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo]: Oh. I did this 3 weeks ago.

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo]: Ah! To avoid a name clash, see https://github.com/modelica/ModelicaStandardLibrary/pull/3091/files#diff-591434336330c10e1295030fd06a33c8R94.

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo]: I see now. There will be a name conflict to a fixRotation instance. OK for me.
